estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class EstatePropertyTag(models.Model):
+    _name = 'estate.property.tag'
+    _description = 'Estate property tag model'
```

Comment:
```suggestion
    _description = 'Estate property tag'
```
General comment: No need of extra info.

Reply:
Changes done!